src/simulation/Simulation.cpp

```diff
@@ -596,6 +596,11 @@ bool Simulation::FloodFillPmapCheck(int x, int y, int type)
 		return TYP(pmap[y][x]) == type;
 }
 
+CoordStack& getCoordStackSingleton () {
```

Comment:
I think you missed a Simulation:: here
Doesn't change anything, really

Reply:
Oh boy, I was in a bit of a hurry and sent the pull request without really checking my code. I'm sorry. It has been fixed.